token_server/token_server.js

```diff
@@ -104,6 +104,7 @@ function generateTokenPeriodically() {
   createServer(async (req, res) => {
     const origin = req.headers['origin'];
     if (!allowedOrigins.has(origin)) {
+      console.warn('Bad origin', origin);
```

Comment:
Did you mean to leave this in? Fine by me if so

Reply:
yeah, thought it would be helpful for future issues, even though it wasn't the cause here.